src/waypoint.jsx

```diff
@@ -23,6 +23,30 @@ function debugLog() {
   console.log(arguments); // eslint-disable-line no-console
 }
 
+let isResizeEventListenerAdded = false;
```

Comment:
I realized that I don't actually need this, because I can use `size` from the Set. I'll fix this if folks are generally okay with this direction. I forgot to set it back to false in the remove event listener function anyway.